src/module/actor/sheet/base.js

```diff
@@ -74,7 +74,7 @@ export class ActorSheetSFRPG extends ActorSheet {
             config: CONFIG.SFRPG
         };
 
-        data.items = this.actor.items;
+        data.items = [...this.actor.items];
```

Comment:
Unfortunately this won't work. `actor.items` has been turned into a `Map`, so this is going to create an array of array's instead of an array of `Item`s. What you need to do is call the `Map.prototype.values()` method, so change it to:
```js
data.items = [...this.actor.items.values()];
```

Reply:
Oops, assumed `Map`s would spread as they were